packages/core/admin/admin/src/content-manager/components/RelationInputDataManager/RelationInputDataManager.js

```diff
@@ -31,15 +31,12 @@ export const RelationInputDataManager = ({
   const { formatMessage } = useIntl();
   const { connectRelation, disconnectRelation, loadRelation, modifiedData, slug, initialData } =
     useCMEditViewDataManager();
-  const [{ query }] = useQueryParams();
-
   const { relations, search, searchFor } = useRelation(`${slug}-${name}-${initialData?.id ?? ''}`, {
     relation: {
       enabled: get(initialData, name)?.count !== 0 && !!endpoints.relation,
       endpoint: endpoints.relation,
       pageParams: {
         ...defaultParams,
```

Comment:
I might be missing something but I understood that `defaultParams` should always be populed with `locale` for relations? Asking because I get an empty object

Reply:
mmh I used `Address` and `Category`, and `Address` doesn't have localization enabled indeed - I thought "_there is no way to not localize a relational field_" meant that they would be localized anyway but I probably didn't understand that right, my bad